src/view/logging_mode/logging_widget.py

```diff
@@ -80,6 +80,8 @@ def new_log_message(self, message: str) -> None:
         """handle incoming log messages"""
         message_dict = json.loads(message)
         level = message_dict["level"]
+        if level == "WARNING":
+            level = "WARN"
```

Comment:
This should not be required though.

Reply:
stimmt warum ist es da?

Reply:
I just deleted this pice of code again. It turned out that the branch was out of date and did not include the logging fix that was already upstream.